routes/stripe.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const stripeRouter = express.Router();
+const STRIPE_SECRET_KEY = process.env.STRIPE_SECRET_KEY;
+const stripe = require('stripe')(STRIPE_SECRET_KEY);
+
+
+stripeRouter.post("/pay", async (req, res, next) => {
+ 
+try {
+  const charge = await stripe.charges.create(req.body);
+  res.send({ success: charge });
+
+  console.log('CHARGE ->', charge)
+
+  res.send({charge});
+} catch (error) {
+  throw error
```

Comment:
Should we "next" the errors so they get sent to the front end via the error handling route?